docs/source/robots.txt

```diff
@@ -7,3 +7,5 @@ Allow: /en/0.19.1/
 Allow: /en/0.19.0/
 Allow: /en/0.18.14/
 Allow: /en/0.17.7/
+Allow: /projects/kedro-viz/en/stable/
+Allow: /projects/kedro-datasets/en/kedro-datasets-2.1.0/
```

Comment:
what about `latest` so at least it's dynamic?

Reply:
Yeah I agree we should be consistent in this file and continue to specify versions we allow, in this case

```
https://docs.kedro.org/projects/kedro-viz/en/v8.0.1/
```

@datajoely we don't want folks to land on `/latest` docs that represent unreleased versions

Reply:
Actually it's not `v8.0.1` _instead_ of `stable`, but _on top of_ (@DimedS pointed out to me in private)

So we'd need to have

viz/stable
viz/8.0.1
datasets/stable
datasets/2.1.0

Does that make sense?

Reply:
Opened #3741 to continue discussing about this